client/flutter/lib/l10n/intl_en_US.arb

```diff
@@ -45,20 +60,17 @@
-  "travelAdviceListOfItemsPageListItem4": "General recommendations for all travellers include…",
-  "travelAdviceListOfItemsPageListItem5": "Wash your hands frequently",
-  "travelAdviceListOfItemsPageListItem6": "Avoid touching your eyes, mouth and nose",
-  "travelAdviceListOfItemsPageListItem7": "Cover your mouth and nose with your bent elbow or tissue when you cough or sneeze",
-  "travelAdviceListOfItemsPageListItem8": "Stay more than 1 meter (3 feet) away from a person who is sick",
-  "travelAdviceListOfItemsPageListItem9": "Follow proper food hygiene practices",
-  "travelAdviceListOfItemsPageListItem10": "Only wear a mask if you are ill with COVID-19 symptoms (especially coughing) or looking after someone who may have COVID-19",
-  "travelAdviceListOfItemsPageListItem11": "Travellers returning from affected areas should:",
-  "travelAdviceListOfItemsPageListItem12": "Self-monitor for symptoms for 14 days and follow national protocols of receiving countries. Some countries may require returning travellers to enter quarantine",
-  "travelAdviceListOfItemsPageListItem13": "Thermal scanners CAN detect if people have a fever but CANNOT detect whether or not someone has the coronavirus",
-  "travelAdviceListOfItemsPageListItem14": "If symptoms occur, such as fever, or cough or difficulty breathing, travellers are advised to contact local health care providers, preferably by phone, and inform them of their symptoms and their travel history",
+  "travelAdviceContainerText": "WHO continues to advise against the application of travel or trade restrictions to countries experiencing COVID-19 outbreaks. It is prudent for travellers who are sick to delay or avoid travel to affected areas, in particular for elderly travellers and people with chronic diseases or underlying healh conditions. “Affected areas” are considered those countries, provinces, territories or cities experiencing ongoing transmission of COVID-19, in contract to areas reporting only imported cases.",
+  "travelAdvicePageListTitle": "Travellers returning from affected areas should:",
+  "travelAdvicePageListItem1Text": "Self-monitor for symptoms for 14 days and follow national protocols of receiving countries",
+  "travelAdvicePageListItem1Image": "assets/travel_advice/self-monitor.png",
+  "travelAdvicePageListItem2Text": "Some countries may require returning travellers to enter quarantine.",
+  "travelAdvicePageListItem2Image": "assets/travel_advice/quarantine.png",
+  "travelAdvicePageListItem3Text": "If symptoms occur, travellers are advsed to contact local health care providers, preferably by phone, and inform them of their symptoms and travel history.",
+  "travelAdvicePageListItem3Image": "assets/travel_advice/doctor.png",
+  "travelAdvicePageButtonGeneralRecommendations": "General\nRecommendations",
+  "travelAdvicePageButtonGeneralRecommendationsLink": "https://who.int",
```

Comment:
https://www.who.int/emergencies/diseases/novel-coronavirus-2019 everywhere

Reply:
is it possible to treat the url fixes as a separate PR? all I did there was extract what's inline and cannot be translated, but needs to be. URLs won't be touched by translators and are only put there so we have the ability to change them to direct to pages localised in the right language.

Reply:
SG!